Moves the "Emergency Call" button to the middle of the lock screen and the "Cancel" button to the left of the zero.
Index: /usr/palm/applications/com.palm.app.phone/app/views/pin/pin-scene.html
===================================================================
--- .orig/usr/palm/applications/com.palm.app.phone/app/views/pin/pin-scene.html
+++ /usr/palm/applications/com.palm.app.phone/app/views/pin/pin-scene.html
@@ -29,7 +29,7 @@
           <div class="key glassbutton nine" id="nine_button" x-mojo-touch-feedback="immediate"><div></div></div>
         </div>
         <div class="keypad-row last">
-          <div class="key glassbutton empty"><div></div></div>
+          <div class="key glassbutton empty" id="cancel_pin" x-mojo-touch-feedback="immediate"><div class="patch-cancel" x-mojo-loc=''><br />Cancel</div></div>
           <div class="key glassbutton zero" id="zero_button" x-mojo-touch-feedback="immediate"><div></div></div>
           <div class="key glassbutton pin-backspace" id="backspace" x-mojo-touch-feedback="immediate"><div></div></div>
         </div>
@@ -39,9 +39,6 @@
       <div id="emergency_pin" x-mojo-touch-feedback="immediate" class="pin-menu-button half" x-mojo-loc=''>
         Emergency Call
       </div>
-      <div id="cancel_pin" x-mojo-touch-feedback="immediate" class="pin-menu-button half" x-mojo-loc=''>
-        Cancel
-      </div>
     </div>
   </div>
 </div>
Index: /usr/palm/applications/com.palm.app.phone/stylesheets/pinpad.css
===================================================================
--- .orig/usr/palm/applications/com.palm.app.phone/stylesheets/pinpad.css
+++ /usr/palm/applications/com.palm.app.phone/stylesheets/pinpad.css
@@ -69,6 +69,12 @@ div.pin-key.pin-glassbutton {
 		bottom: 0px;
 		width: 320px;
 		height: 60px;
+		margin-left: 80px;
+}
+
+.patch-cancel {
+		font-size: 16px;
+		line-height: 15px;
 }
 
 .pin-menu-button {
